README.md

```diff
@@ -17,7 +17,7 @@ To learn more, see the [Cluster API KEP][cluster-api-kep].
 ## Get involved!
 
 * Join our Cluster API working group sessions
-  * Weekly on Wednesdays @ 11:00 PT (19:00 UTC) on [Zoom][zoomMeeting]
+  * Weekly on Wednesdays @ 10:00 PDT (17:00 UTC) on [Zoom][zoomMeeting]
```

Comment:
is this something we plan to change everytime we switch between pdt/pst?
the time should be 10:00, but should the time zone be just PT to say it is applies whether it is daylight savings or not?
not sure how we would make utc indifferent to daylight savings.

Reply:
Kenny are you saying that we switched the meeting time in Seattle from 11:00 to 10:00 as a result of daylight savings time changes?

Reply:
The concern is around changing PT to PDT and the correct UTC time is dependent on whether it is PDT or PST.
The UTC time will be incorrect whenever PDT changes to PST
10:00 PDT is 17:00 UTC
10:00 PST is 18:00 UTC.

Reply:
Hello Kenny, I'm more asking about the meeting. Did the meeting time change from 11:00 AM to 10:00 AM or was it always 10:00 AM? If so we should just change it to PST.

And it sounds like that is the point you are making?

Reply:
According to the meeting notes, the meeting time was moved to "10AM PST" starting Dec 20, 2017.

I am just concerned for people using the UTC time, as it does not factor in the daylight savings adjustments.
I would suggest keeping PT or use PDT/PST and note about the UTC being either 17:00 or 18:00 depending on whether daylight savings is being observed or not.

Reply:
Let's change this to 10 AM PST and leave the UTC out of it.

Reply:
Is it worth keeping it as PT, to avoid potential issues with people using PST for conversion.

Reply:
No -- because it will inevitably be wrong when we forget to update it.

Reply:
What of "10:00 PT" that needs updating?
I'm not sure what was the original reason for changing the meeting time, but I don't think it was due to daylight savings, or was it?

@roberthbailey can you comment on whether the changing of the meeting time was due to daylight savings or not.

Reply:
From the definition of PT via google: 

```
PDT = Pacific Daylight Time = fixed to UTC-7. PST = Pacific Standard Time = fixed to UTC-8. PT = Pacific Time = a general reference to the time zone, which alternates between PDT and PST depending on the time of year.
```

IF the meeting is bound to PT, then sure, that is a better choice. I wasn't thinking that was the case.

Reply:
That is the question. As the original time listed is PT, and we kept the 10AM PT time before and after daylight savings began this year, it would seem we are bound to PT.

Reply:
Like most sig meetings, the meeting is fixed in pacific time, not UTC time. So PT is more accurate that PST or PDT.